src/api/Jalasoft.TeamUp.Projects.API/local.settings.json

```diff
@@ -1,8 +1,8 @@
 {
   "IsEncrypted": false,
   "Values": {
-    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "AzurJobsStorage": "UseDevelopmentStorage=true",
```

Comment:
did you mean to change this?

Reply:
my bad, was a typo, sorry, it is fixed now